recipe/meta.yaml

```diff
@@ -41,12 +41,15 @@ requirements:
     - xsimd  # [cuda_compiler_version != "10.2" or not linux] 
     - numpy
     - pandas
+    - dcmtk
+    - fmjpeg2koj
     - m2-patch  # [win]
 
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - pandas
+    - fmjpeg2koj
```

Comment:
you may want to consider adding a line like "'run_exports" to the package `fmjpeg2koj`

For example, you can look at the openjpeg pin:

https://github.com/conda-forge/openjpeg-feedstock/blob/main/recipe/meta.yaml#L16


Then we can add `fmjpeg2koj` to the global pinnigns, and ensure version consistency accross the ecosystem.